stackable-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.7@sha256:3e1adcc31c6073d010b8043b070bd089d7bf37ee2c397c110211a6273453433f
+FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:621f5245fb3e8597a626163cdf1229e1f8311e07ab71bb1e9332014b51c59f9c 
```

Comment:
Could you add a comment with the tag/version (not just the digest) ? This will affect all builds (amd too) and it will be a reminder of what exactly changed here

Reply:
Yes, done. Also updated the changelog.